reporting/impl/src/main/java/org/jboss/windup/reporting/freemarker/problemsummary/GetProblemSummariesMethod.java

```diff
@@ -99,6 +112,10 @@ public int compare(String severity1, String severity2)
         ClassificationService classificationService = new ClassificationService(context);
         for (ClassificationModel classification : classificationService.findAll())
         {
+            Set<String> tags = classification.getTags();
+            if (!TagUtil.isTagsMatch(tags, includeTags, excludeTags, false))
```

Comment:
I'm not sure if "validate()" is a good term for what this is doing, though. It is more like a TagFilter to me.